### PR TITLE
Provide page id and revision timestamp in revision data end point

### PR DESCRIPTION
**Pull request by JonCook**
Relating to changes described in:
https://phabricator.wikimedia.org/T95364

### CODE REVIEW

**Comment by gwicke** [comment]
@JonCook, thank you for your patch! This looks good to go to me. The only blocker is that we'll need schema migration support for existing content before we can merge / deploy this. @eevans is currently working on https://phabricator.wikimedia.org/T75808, and is looking into support for adding columns in particular.

File: mods/page_revisions.js [hunk 1]
```diff
@@ -193,6 +196,10 @@ PRS.prototype.fetchAndStoreMWRevision = function (restbase, req) {
         var restrictions = Object.keys(apiRev).filter(function(key) {
             return /hidden$/.test(key);
         });
+
+        //get the redirect property, it's inclusion means true
+        var redirect = typeof dataResp.redirect !== "undefined" ? true : false;
```
Comment (d00rman) [inline mods/page_revisions.js]: Can be simply:

```javascript
var redirect = dataResp.redirect !== undefined;
```

File: test/features/pagecontent/revisions.js [hunk 2]
```diff
@@ -12,6 +12,7 @@ describe('revision requests', function() {
 
 	var revOk = 642497713;
 	var revDeleted = 645504917;
+    var revRedirect = 591082967;
```
Comment (d00rman) [inline test/features/pagecontent/revisions.js]: Indentation

**Comment by d00rman** [comment]
Looks really good @JonCook, thank you for contributing! As @gwicke mentioned, we need to implement and deploy schema migrations before we can deploy this change, so we are leaving it open until then. Minor nitpicks in-lined.

**Comment by JonCook** [comment]
Thanks for the comments. I'll fix the minor nitpicks later. The indentation is a bit strange as well as it looks ok in my text editor.

**Comment by d00rman** [comment]
> The indentation is a bit strange as well as it looks ok in my text editor.

Might be a tabs vs spaces issue

**Comment by gwicke** [comment]
> Might be a tabs vs spaces issue

Yeah, there are still some stray tabs in some of the test files. Lets clean that up in a separate patch.

**Comment by JonCook** [comment]
I updated the minor nitpicks. Hope it is ok. Think it is ready to go now when you've finished the schema migration changes.

**Comment by gwicke** [comment]
@JonCook, the schema support has landed now,  but we'll hold of with merging this until after the Monday deploy to reduce the amount of change we deploy at once.

**Comment by JonCook** [comment]
@gwicke Did the Monday deploy go ok? Is it safe for you guys to go ahead with this merge?

**Comment by gwicke** [comment]
@JonCook, the Monday deploy went well, and I just merged the capability to actually add columns. We'll be doing one more deploy tomorrow before likely pushing this out on Thursday.

**Comment by gwicke** [comment]
@JonCook, after some testing your code went live 15 minutes ago. It's all looking great, so thanks a lot for your contribution! The next step will now be to leverage this information in HTML dumps (https://phabricator.wikimedia.org/T93396 and https://github.com/wikimedia/htmldumper).

**Comment by JonCook** [comment]
Great, thanks. Glad I could help/contribute!

**Pull request merged**